codegen/src/test/resources/q10/q10.json

```diff
@@ -0,0 +1,181 @@
+    ],
+    "RelationProcessFunction": [
+      {
+        "name": "Q10lineitem",
+        "relation": "lineitem",
+        "this_key": [
+          "orderkey" 
+        ],
+        "next_key": [
+          "custkey", 
+          "c_name",
+          "c_acctbal",
+          "c_phone",
+          "c_address",
+          "c_comment",
+          "n_name"
+        ],
+        "child_nodes": 3,
+        "is_Root": true,
+        "is_Last": true,
+        "rename_attribute": null,
+        "select_conditions": {
+          "operator": "&&",
+          "values": [
+            {
+              "left_field": {
+                "type": "attribute",
+                "relation": "lineitem",
+                "name": "returnflag"
+              },
+              "right_field": {
+                "type": "constant",
+                "var_type": "varchar",
+                "value": "R"
+              },
+              "operator": "=="
+            }
+          ]
+        }
+      },
+      {
+        "name": "Q10Orders",
+        "relation": "orders",
+        "this_key": [
+          "custkey"
+        ],
+        "next_key": [
+          "orderkey"
+        ],
+        "child_nodes": 2,
+        "is_Root": false,
+        "is_Last": true,
+        "rename_attribute": null,
+        "select_conditions": {
+          "operator": "&&",
+          "values": [
+            {
+              "left_field": {
+                "type": "attribute",
+                "relation": "orders",
+                "name": "orderdate"
+              },
+              "right_field": {
+                "type": "constant",
+                "var_type": "Date",
+                "value": "1993-10-01"
+              },
+              "operator": ">="
+            },
+            {
+                "left_field": {
+                  "type": "attribute",
+                  "relation": "orders",
+                  "name": "orderdate"
+                },
+                "right_field": {
+                  "type": "constant",
+                  "var_type": "Date",
+                  "value": "1994-01-01"
+                },
+                "operator": "<"
+              }
+          ]
+        }
+      },
+      {
+        "name": "Q10Customer",
+        "relation": "customer",
+        "this_key": [
+          "custkey"
+        ],
+        "next_key": [
+          "custkey"
+        ],
+        "child_nodes": 1,
+        "is_Root": false,
+        "is_Last": true,
+        "rename_attribute": null,
+        "select_conditions": {}
+      },
+      {
+        "name": "Q10Nation",
+        "relation": "nation",
+        "this_key": [
+          "nationkey"
+        ],
+        "next_key": [
+          "nationkey"
+        ],
+        "child_nodes": 0,
+        "is_Root": false,
+        "is_Last": true,
+        "rename_attribute": null,
+        "select_conditions": {}
+      }
+    ],
+    "AggregateProcessFunction": [
+      {
+        "name": "Q10Aggregate",
+        "this_key": [
```

Comment:
this_key here should be all attributes in the group_by clause.

Reply:
revised in [https://github.com/DanishAlsayed/gui-codegen/pull/26/commits/9a8adc0d4669d8d2114450b9a177fd4ed20ff067](url)